tests/api/search/search.test.coffee

```diff
@@ -45,6 +45,28 @@ describe 'search:global', ->
 
         return
 
+    describe 'filter', ->
+      it 'should reject an invalid filter', (done)->
+        search 'humans', 'yo', 'foo'
+        .then undesiredRes(done)
+        .catch (err)->
+          err.statusCode.should.equal 400
+          err.body.status_verbose.should.startWith 'invalid filter: foo'
+          done()
+        .catch undesiredErr(done)
+
+        return
+
+      it 'should accept a filter parameter', (done)->
```

Comment:
Would be more complete to test with a fixtured entity with a wd label 
`author = createHuman { labels: { en: 'Gilles Deleuze' } }`
and check its presence 
`search 'humans', 'Gilles Deleuze', 'inv'`
`_.map(results, 'uri').includes(author.uri).should.be.true()`
and its absence 
`search 'humans', 'Gilles Deleuze', 'wd'`
`_.map(results, 'uri').includes(author.uri).should.be.false()`
well doesnt have to be Gillou, could be a random label

Reply:
yep, that was definitely a good idea: the test was passing but no search was done ^^ patch incoming

Reply:
done